cloud/endagaweb/views/dashboard.py

```diff
@@ -688,6 +689,12 @@ def _handle_request(self, request):
             request.session['end_date'] = request.POST.get('end_date', None)
             request.session['services'] = request.POST.getlist('services[]',
                                                                None)
+            # Added to check password to download the csv
```

Comment:
This check is written inside ActivityView only.For post request of ActivityView only  it will check password in request and authorize the password.